src/SkillService.Validation/CreateSkillValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.SkillService.Data.Interfaces;
+using LT.DigitalOffice.SkillService.Validation.Interfaces;
+
+namespace SkillService.Validation
```

Comment:
this namespace doesn't correct. is should be like LT.DigitalOffice.SkillService.Validation.Skill